src/gui/buttons.py

```diff
@@ -23,6 +25,45 @@ def command_bar(self) -> list[UIFlatButton]:
         ...
 
 
+class PixelatedNinePatch(NinePatchTexture):
+    """
+    This only exists because I can't see any way to set the pixelated flag to True via the base NinePatchTexture.
+    The implementation of draw_sized here is literally identical, the only difference is the default for the pixelated param.
+    """
+
+    def draw_sized(
+        self,
+        *,
+        position: tuple[float, float] = (0, 0),
+        size: tuple[float, float],
+        pixelated: bool = True,
+        **kwargs
+    ):
+        """
+        Draw the 9-patch.
+
+        :param size: size of the 9-patch
+        """
+        # TODO support to draw at a given position
+        self.program.set_uniform_safe(
+            "texture_id", self._atlas.get_texture_id(self._texture.atlas_name)
+        )
+        if pixelated:
+            self._atlas.texture.filter = self._ctx.NEAREST, self._ctx.NEAREST
+        else:
+            self._atlas.texture.filter = self._ctx.LINEAR, self._ctx.LINEAR
+
+        self.program["position"] = position
+        self.program["start"] = self._left, self._bottom
+        self.program["end"] = self.width - self._right, self.height - self._top
+        self.program["size"] = size
+        self.program["t_size"] = self._texture.size
+
+        self._atlas.use_uv_texture(0)
+        self._atlas.texture.use(1)
+        self._geometry.render(self._program, vertices=1)

```

Comment:
If you ever find yourself doing the exact same thing as one of the base classes, this is the syntax to just defer the call up the inheritance hierarchy:
```suggestion
        super().draw_sized(position=position, size=size, pixelated=pixelated, **kwargs)

```